### PR TITLE
updated upload text for image size/acceptable types

### PR DESCRIPTION
**Pull request by geezacoleman**
Nothing major, just updated the text 'Max file size: 5mb, accepted: jpg|gif|png'

### CODE REVIEW

File: search/src/Components/upload/react-images-upload/index.js [hunk 1]
```diff
@@ -276,7 +276,7 @@ ReactImageUploadComponent.defaultProps = {
   buttonText: "Choose images",
   buttonType: "button",
   withLabel: true,
-  label: "Max file size: 5mb, accepted: jpg|gif|png",
+  label: "Max file size: 5 MB | File types accepted: .jpg, .gif, .png",
```
Comment (jnothman) [inline search/src/Components/upload/react-images-upload/index.js:279]: (I think we also permit TIF)

Comment (geezacoleman) [reply search/src/Components/upload/react-images-upload/index.js]: Just added the extra text

**Comment by geezacoleman** [comment]
@jnothman ok to merge and close this one?

**Review by jnothman — APPROVED** [review]

**Comment by jnothman** [comment]
Thanks @geezacoleman

**Pull request merged**